scripts/predict.py

```diff
@@ -1,46 +1,62 @@
 import json
 import os
+import re
 import pickle
 import sys
 from os.path import dirname, abspath
 sys.path.append(dirname(dirname(abspath(__file__))))
 from pkspace.utils.loaders import PKSpaceLoader # noqa
+from configapp.default_settings import IMAGE_SUFFIX # noqa
```

Comment:
@Adman I do not like this too much -- this is leaking data and making an advantage of the fact that both this script and the settings are stored as Python files/modules.

I believe the better approach here would be to have a command line argument for the value of `IMAGE_SUFFIX`.